packages/suite/src/components/suite/modals/AddAccount/components/EnableNetwork.tsx

```diff
@@ -1,50 +1,21 @@
-import React, { useState } from 'react';
+import React from 'react';
```

Comment:
One more thing, it should be green when enabled.

https://user-images.githubusercontent.com/33235762/189847420-5608baf0-21c7-4dad-bc63-f7e9b22c6aa4.mov